core-bundle/src/Csrf/MemoryTokenStorage.php

```diff
@@ -14,11 +14,12 @@
 
 use Symfony\Component\Security\Csrf\Exception\TokenNotFoundException;
 use Symfony\Component\Security\Csrf\TokenStorage\TokenStorageInterface;
+use Symfony\Contracts\Service\ResetInterface;
 
-class MemoryTokenStorage implements TokenStorageInterface
+class MemoryTokenStorage implements TokenStorageInterface, ResetInterface
 {
     /**
-     * @var array
+     * @var array|null
```

Comment:
This fix is wrong. The `reset()` method should use `$this->tokens = [];` instead, shouldn't it?

Reply:
no, the uninitialized state is null, the property does not have a default value

Reply:
You are right. Sorry.